CHANGELOG.md

```diff
@@ -7,7 +7,26 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [7.0.12] - 2023-11-16
 
-- Adds Phone Number normalisation
+In this release, the core API routes have been updated to incorporate phone number normalization before processing. Consequently, existing entries in the database also need to undergo normalization. To facilitate this, we have included a migration script to normalize phone numbers for all the existing entries.
```

Comment:
@rishabhpoddar Wasn't sure how this migration script will be executed. Expecting some changes here.